pubsub-service-api.yml

```diff
@@ -480,30 +469,31 @@ components:
           - data
         properties:
           from:
-            description: the Peer Id of the message's sender
+            description: "The peer ID of the sender."
             type: string
             example: "QmcXhKGEAJT9DwwzKeEZsWrYTTGkzBdgeKif2MdAXBswJv"
           data:
-            description: the base64 encoded message's content,
+            description: "The base64-encoded content of the message."
             type: string
             example: "YWJj"
           pubkey:
-            description: the public key associated to the sender's peer ID, formatted as a base64-encoded x509 DER public key
+            description: "The sender's public key, formatted as a base64-encoded X.509 DER public key."
```

Comment:
I think DER is correct (as it works for ECDSA and RSA) but we should specify the base64 alphabet.